### PR TITLE
Instrument model

### PR DESCRIPTION
**Pull request by antgonza**
This adds the instrument model field as a required field for EBI submission. Most of the changes are due to fixing tests. Notable changes: changed the template documentation page, and travis.yml to remove two lines and make it one (suggestion by @josenavas in previous request and found how to do it).

### CODE REVIEW

**Comment by antgonza** [comment]
This is ready for review: @biocore/qiita-dev

File: qiita_ware/ebi.py [hunk 1]
```diff
@@ -182,14 +195,25 @@ def __init__(self, preprocessed_data_id, action):
...
             self.sample_demux_fps[k] = get_output_fp("%s.fastq.gz" % k)
 
         if nvp:
-            error_msgs.append("These samples do not have a valid platform: "
-                              "%s" % (', '.join(nvp)))
+            error_msgs.append("These samples do not have a valid platform "
+                              "(instrumet model wasn't checked): %s" % (
+                                ', '.join(nvp)))
+        if nvim:
+            error_msgs.append("These samples have ado not have a valid "
```
Comment (ElDeveloper) [inline qiita_ware/ebi.py]: ado -> do

reminded me of a song though ... :musical_score:

Comment (ElDeveloper) [inline qiita_ware/ebi.py]: oh and have is before the `not`.

File: qiita_ware/ebi.py [hunk 2]
```diff
@@ -182,14 +195,25 @@ def __init__(self, preprocessed_data_id, action):
...
             self.samples_prep[k] = sample_prep
             self.sample_demux_fps[k] = get_output_fp("%s.fastq.gz" % k)
 
         if nvp:
-            error_msgs.append("These samples do not have a valid platform: "
-                              "%s" % (', '.join(nvp)))
+            error_msgs.append("These samples do not have a valid platform "
+                              "(instrumet model wasn't checked): %s" % (
+                                ', '.join(nvp)))
+        if nvim:
```
Comment (ElDeveloper) [inline qiita_ware/ebi.py]: Can you add a test case in `test_ebi.py` for the cases where there are invalid instrument models?

File: qiita_pet/support_files/doc/source/tutorials/prepare-templates.rst [hunk 3]
```diff
@@ -85,19 +85,31 @@ Required fields for EBI submission
...
-| ``experiment_design_description``   | free text                                 | High-level description of the study (for example, *A longitudinal study of the gut microbiome of two human subjects*).                                                                                                              |
-+-------------------------------------+-------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++-------------------------------------+-------------------------------------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/tutorials/prepare-templates.rst]: I can't tell from the GitHub interface, but there seems to be something off with the formatting of this table when you build the docs:

screen shot 2015-10-02 at 3 15 42 pm

**Comment by ElDeveloper** [comment]
I think this looks good overall, I would just suggest adding a note to the ChangeLog about this new required column for submissions to EBI.

**Comment by josenavas** [comment]
:+1: once @ElDeveloper comments are addressed. There will be some merge conflicts with my PR, but it is way easier to solve them in my branch.

**Comment by antgonza** [comment]
Ready to go!

**Comment by josenavas** [comment]
:+1: @ElDeveloper can you review/merge?

**Pull request merged**